spec/index.bs

```diff
@@ -671,15 +671,31 @@ initially empty.
 
     1. Set |localName| to |tagName| in [=ASCII lowercase=].
     1. If |elementNs| is an empty string, set |elementNs| to [=HTML namespace=].
+    1. If |property| starts with "on", then return {{TrustedScript}}.
```

Comment:
Need to remove this step from getPropertyType because it's only attributes that can be event handlers